octo/run-integration-tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+richgo test ./integration/...
```

Comment:
```suggestion
richgo test ./integration/... -run Integration
```
I think we should do this.

It will limit integration test runs to those that are in the `./integration` folder and it'll also filter on the test name so that you need something like `TestIntegration` or `FooIntegration`. The run filter is probably not necessary but gives it a little extra flexibility.